Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.h

```diff
@@ -53,7 +53,7 @@ enum class PickerDismissalReason : uint8_t;
 - (NSArray<NSString *> *)currentAvailableActionTitles;
 - (NSArray<NSString *> *)acceptedTypeIdentifiers;
 
-+ (std::pair<RetainPtr<NSURL>, RetainPtr<NSURL>>)_copyToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager;
++ (std::tuple<BOOL, RetainPtr<NSURL>, RetainPtr<NSURL>>)_moveToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager asCopy:(BOOL)asCopy;
```

Comment:
maybe instead of a tuple, a new structure type should just be used instead for clarity

Reply:
+1 to a C++ struct (`std::optional<std::pair<…>>` is also a viable alternative)